### PR TITLE
Packet single platform display in ci/e2e

### PR DESCRIPTION
**Pull request by bhaskarhc**
- 3 packet pipelines will run in different kubernetes versions .
- ui changes .
- images are updated with new images .

Signed-off-by: bhaskarjb <hcbhaskar7@gmail.com>

screenshot from 2019-03-01 18-49-23

### CODE REVIEW

File: src/app/services/agile.services.ts [hunk 1]
```diff
@@ -15,11 +15,11 @@ export class AgileService {
...
     this.host = window.location.host;
     if (this.host == "localhost:4200") {
-      this.apiurl = "http://localhost:3000/";
+      this.apiurl = "https://localhost:3000/";
     } else if (this.host == "staging.openebs.io") {
-      this.apiurl = "https://staging.mayadata.io/api/";
+      this.apiurl = "https://workloads.openebs.ci/";
     } else {
-      this.apiurl = "https://openebs.io/api/";
+      this.apiurl = "https://workloads.openebs.ci/";
```
Comment (chandankumar4) [inline src/app/services/agile.services.ts]: @inyee786 Can you verify this agile service url

File: src/app/services/agile.services.ts [hunk 2]
```diff
@@ -15,11 +15,11 @@ export class AgileService {
   constructor(private http: HttpClient) {
     this.host = window.location.host;
     if (this.host == "localhost:4200") {
-      this.apiurl = "http://localhost:3000/";
+      this.apiurl = "https://localhost:3000/";
```
Comment (chandankumar4) [inline src/app/services/agile.services.ts]: https -> http

File: src/app/services/kubernetes.service.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ export class KubernetsService {
     constructor(private http: HttpClient) {
         this.host = window.location.host;
         if ((this.host.toString().indexOf("localhost") + 1) && this.host.toString().indexOf(":")) {
-            this.apiurl = "http://localhost:3000/";
+            this.apiurl = "https://localhost:3000/";
```
Comment (chandankumar4) [inline src/app/services/kubernetes.service.ts]: https -> http

File: src/app/services/savereaddelete.service.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ export class PersonService {
   constructor(private http: HttpClient) {
     this.host = window.location.host;
     if ((this.host.toString().indexOf("localhost") + 1) && this.host.toString().indexOf(":")) {
-      this.apiurl = "http://localhost:3000/";
+      this.apiurl = "https://localhost:3000/";
```
Comment (chandankumar4) [inline src/app/services/savereaddelete.service.ts]: https -> http

File: src/app/services/agile.services.ts (hunk 1 above)
Comment (inyee786) [reply src/app/services/agile.services.ts]: this is correct url "https://openebs.io/api/

File: src/app/workloads/workloads.component.html [hunk 5]
```diff
@@ -65,13 +65,13 @@
             <br>
             <div class="col-md-12 d-flex justify-content-around workloads my-3">          
               <div class="workload-deatils mx-2 divideLine text-secondary w-33"  *ngFor="let pvc of pvcarray">
-                  <img src="/assets/images/workloads-logos/{{workloadName}}.svg" height="28" alt="">
+                  <img src="/assets/images/workloads-logos/{{workloadName}}.png" height="30" alt="">
```
Comment (inyee786) [inline src/app/workloads/workloads.component.html]: did all the image are png??

**Comment by inyee786** [comment]
@bhaskarhc why you have delete the svg file and put png??

File: src/app/workloads/workloads.component.html (hunk 5 above)
Comment (bhaskarhc) [reply src/app/workloads/workloads.component.html]: Yes ..! All images arein png

**Comment by bhaskarhc** [comment]
These all png images are updated one used in ‘openebs.io’

**Comment by inyee786** [comment]
the png image is not coming out well in dashboard

File: src/app/workloads/workloads.component.html (hunk 5 above)
Comment (inyee786) [reply src/app/workloads/workloads.component.html]: but elastic search image is svg

File: src/app/services/agile.services.ts [hunk 6]
```diff
@@ -17,9 +17,9 @@ export class AgileService {
     if (this.host == "localhost:4200") {
       this.apiurl = "http://localhost:3000/";
     } else if (this.host == "staging.openebs.io") {
-      this.apiurl = "https://staging.mayadata.io/api/";
+      this.apiurl = "http://workloads.openebs.ci/";
     } else {
-      this.apiurl = "https://openebs.io/api/";
+      this.apiurl = "http://workloads.openebs.ci/";
```
Comment (inyee786) [inline src/app/services/agile.services.ts]: http => https

File: src/app/services/agile.services.ts [hunk 7]
```diff
@@ -17,9 +17,9 @@ export class AgileService {
     if (this.host == "localhost:4200") {
       this.apiurl = "http://localhost:3000/";
     } else if (this.host == "staging.openebs.io") {
-      this.apiurl = "https://staging.mayadata.io/api/";
+      this.apiurl = "http://workloads.openebs.ci/";
```
Comment (inyee786) [inline src/app/services/agile.services.ts]: http => https

File: src/app/services/kubernetes.service.ts [hunk 8]
```diff
@@ -13,7 +13,7 @@ export class KubernetsService {
         if ((this.host.toString().indexOf("localhost") + 1) && this.host.toString().indexOf(":")) {
             this.apiurl = "http://localhost:3000/";
         } else {
-            this.apiurl = "https://workloads.openebs.ci/";
+            this.apiurl = "http://workloads.openebs.ci/";
```
Comment (inyee786) [inline src/app/services/kubernetes.service.ts]: http => https

File: src/app/services/litmus.services.ts [hunk 9]
```diff
@@ -15,7 +15,7 @@ export class LitmusService {
     ) {
       this.apiurl = "http://localhost:3000/";
     } else {
-      this.apiurl = "https://workloads.openebs.ci/";
+      this.apiurl = "http://workloads.openebs.ci/";
```
Comment (inyee786) [inline src/app/services/litmus.services.ts]: http => https

File: src/app/services/savereaddelete.service.ts [hunk 10]
```diff
@@ -18,7 +18,7 @@ export class PersonService {
     if ((this.host.toString().indexOf("localhost") + 1) && this.host.toString().indexOf(":")) {
       this.apiurl = "http://localhost:3000/";
     } else {
-      this.apiurl = "https://workloads.openebs.ci/";
+      this.apiurl = "http://workloads.openebs.ci/";
```
Comment (inyee786) [inline src/app/services/savereaddelete.service.ts]: http => https

File: src/app/workloads/workloads.component.html (hunk 5 above)
Comment (bhaskarhc) [reply src/app/workloads/workloads.component.html]: elastic search png also there

File: src/app/services/litmus.services.ts (hunk 9 above)
Comment (bhaskarhc) [reply src/app/services/litmus.services.ts]: done

File: src/app/services/agile.services.ts (hunk 1 above)
Comment (bhaskarhc) [reply src/app/services/agile.services.ts]: done

**Comment by chandankumar4** [comment]
`packet` text somewhere is in caps, can you make the text similar and use the tag specific name in place of  
packet1, packet2 and packet3

**Pull request merged**